[zh_tw] Re-translate zh_tw.json file (Update to 0.0.2-alpha.100)

Jimmy-sheep opened the pull request:
Re-translate all strings.
Thanks for [Incompleteusern](https://github.com/Incompleteusern)'s review and help.

----- Translation Notes -----
1. Some of text is translated basic on [Microsoft product translation](https://www.microsoft.com/zh-tw/language/Search?&langID=129).
2. As the punctuation in Unicode font is also halfwidth in Minecraft, so I prefer Ascii font than Unicode font in translate.
3. Translate `Hades` into `社交監控`, this more conform to it's function.
4. The strings that have proper noun(ex: Item Rarities, `Perfect` prifix or Item `Corkian Amplifier`) I only translate the `xxx.name`, and keep `xxx.description` in English to prevent user being confuse, if need translates please let me know.

----- Need Confirm -----
1. Not sure what's the Packet means in `"feature.wynntils.packetDebugger.name": "Packet Debugger"`, so I translate into `封包偵錯工具` which "packet" means here is as transmittion packet.
2. Not sure what's `"screens.wynntils.chatTabsGui.consuming": "Consuming"` means in game, so I translate into `消耗`, please help check it.

Commit:
[zh_tw] Re-translate zh_tw.json file

kristofbolyai commented:
The json file seems to be not correct? When you fix it, could you please upload this to https://crowdin.com/project/wynntils-mod?

Commit:
Fix missing quotation

Jimmy-sheep commented:
> The json file seems to be not correct? When you fix it, could you please upload this to https://crowdin.com/project/wynntils-mod?

Oh, ok I will upload there after I finish.

Commit:
[zh_tw] Update translation (Part 2)

Commit:
[zh_tw] Update translation (Part 3)

Commit:
Merge branch 'main' into main

Commit:
[zh_tw] Update translation (Part 4)

Commit:
Merge branch 'main' into main

Commit:
Merge branch 'main' into main

kristofbolyai commented:
Are the translations finished? Are you still working on it?

Incompleteusern commented:
> Re-translate all strings.
> 
> The previous translate doesn't make any sense, but still thanks for last translator's try.

Thanks for fixing my translations :)

Jimmy-sheep commented:
> Are the translations finished? Are you still working on it?

Yes, I'm still working on it, I think there's still have about 1/3 more...

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,984 @@
-  "feature.wynntils.abilityRefreshRedirect.name": "重新導向技能刷新話",
-  "feature.wynntils.abilityTreeScroll.invertScroll.description": "技能樹裡面，在滾動時要倒轉滾動方向嗎？",
-  "feature.wynntils.abilityTreeScroll.invertScroll.name": "倒轉滾動",
-  "feature.wynntils.abilityTreeScroll.name": "技能樹滾動",
```
the precedent as far as i know for `ability tree` is `技能樹`

Jimmy-sheep replied:
Actually, `ability`(能力) in Chinese is more like "talent", but `技能`(skill) is more like "magic spell", also only 4 points is magic spell in game, that's why I translate `ability` into `能力`.

Incompleteusern replied:
Fair enough, though some of the abilities are quite magical.

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,984 @@
-  "feature.wynntils.bombBellTracking.name": "炸彈記錄",
-  "feature.wynntils.chatCoordinates.name": "訊息座標",
-  "feature.wynntils.chatItem.name": "訊息物品",
-  "feature.wynntils.chatRedirect.craftedDurability.description": "合成物品的耐久低訊息要如何顯示 How should crafted item durability messages appear？",
```
nice fix!

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,984 @@
+  "feature.wynntils.bombBellTracking.name": "炸彈鈴追蹤",
+  "feature.wynntils.chatCoordinates.name": "聊天座標",
+  "feature.wynntils.chatItem.name": "聊天物品",
+  "feature.wynntils.chatRedirect.craftedDurability.description": "該如何顯示製作的物品耐久度訊息?",
```
this uses `装作` while the following sentence uses `製做`, is this intended?

Jimmy-sheep replied:
Sorry, it's my typo, cause they have same read sound in Chinese. I should type `製作` instead of `製做`.

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,984 @@
+  "feature.wynntils.chatRedirect.speed.name": "速度效果訊息過濾",
+  "feature.wynntils.chatRedirect.toolDurability.description": "該如何顯示採集工具耐久度的訊息?",
+  "feature.wynntils.chatRedirect.toolDurability.name": "採集工具耐久度過濾",
+  "feature.wynntils.chatRedirect.unusedPoints.description": "該如何顯示關於未使用能力點或技能點的訊息?",
```
I had `屬性點` as precedent for skill point, but this might also be fine. I am a bit concerned about using `能力` for both skill points and ability tree earlier though

Jimmy-sheep replied:
You are right, I'll change this into `屬性點`, thanks for review.

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,984 @@
+  "feature.wynntils.chatRedirect.toolDurability.name": "採集工具耐久度過濾",
+  "feature.wynntils.chatRedirect.unusedPoints.description": "該如何顯示關於未使用能力點或技能點的訊息?",
+  "feature.wynntils.chatRedirect.unusedPoints.name": "未使用能力點或技能點訊息篩選",
+  "feature.wynntils.chatTabs.name": "聊天標籤",
```
Chat tabs are more of channels that messages are filtered through than actual tabs/labels

Commit:
Merge branch 'main' into main

Incompleteusern commented:
> Re-translate all strings. Thanks for [Incompleteusern](https://github.com/Incompleteusern)'s review.
> 
> ----- Note -----
> 
> 1. Item rarity `legendary` is almost same as `fabled` in Chinese, so I translate into `史詩`.
> 2. Not sure what is `Dry Boxes` and `Dry Count`, so I directly translate, if anyone know ,please help translate.
> 3. Not sure what is `Last Gamma`, so I directly translate, if anyone know ,please help translate.
> 4. ~Not sure should I translate `Hades` or not, so I keep it in English. (is it a greek god's name?)~
> 5. Translate `Hades` into `社交監控`, this more conform to it's function.

1. Probably fine
2. In Wynncraft you can lootrun to get mythic boxes by opening chests. Dry Count is how many chests you have gone without getting a mythic, Dry Boxes is how many boxes you have found in chests without getting a mythic. When you find a mythic, both of these values are set to zero.
3. Gamma is how bright the ingame world is rendered, last gamma is the most recent gamma we used and doesn't really matter translation, probably translate the same as "Last Wynncraft Pack Hash"
4 & 5. Fine both ways

Jimmy-sheep commented:
> 2\. In Wynncraft you can lootrun to get mythic boxes by opening chests. Dry Count is how many chests you have gone without getting a mythic, Dry Boxes is how many boxes you have found in chests without getting a mythic. When you find a mythic, both of these values are set to zero.

Ahh, this might be hard for me to translate to Chinese @_@. I'll try.... thanks~
Btw, where can I find these string in game? I can't find it in "QuestBook->Configuration"

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,984 @@
-  "feature.wynntils.customBarsOverlay.overlay.manaBankBar.name": "魔力池條",
-  "feature.wynntils.customBarsOverlay.overlay.manaBar.manaTexture.description": "魔力條使用哪樣的材質顯示？",
-  "feature.wynntils.customBarsOverlay.overlay.manaBar.manaTexture.name": "魔力材質",
+  "feature.wynntils.customBarsOverlay.overlay.manaBankBar.name": "魔量條",
```
This might be better as showing that its an additional storage of mana beyond the normal mana bar, currently its a bit similar to `mana bar`'s translation

Jimmy-sheep replied:
Oh, so its additional mana? like...the health we got from golden apple in vanilla minecraft?

Jimmy-sheep replied:
Also, is "blood pool" same kind like this?

Incompleteusern replied:
Yeah sort of, it interacts a bit more specially but its an additional storage for health/mana effectively.

Jimmy-sheep replied:
Thanks, for solving my problem, I'm new player that join wynncraft only 1 month XD

Commit:
[zh_tw] Update translation (Part 5)

Changes some translation, fix some typo, re-translate some string.

Especially thanks for Incompleteusern's help.

Commit:
[zh_tw] Update translation (Part 5.5)

Fix missing string.

Commit:
Merge branch 'main' into main

Commit:
[zh_tw] Update translation (Part 6)

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -145,10 +145,10 @@
+  "feature.wynntils.dataStorage.dryBoxes.description": "函數變量乾燥盒",
+  "feature.wynntils.dataStorage.dryBoxes.name": "無神話物品箱數",
+  "feature.wynntils.dataStorage.dryCount.description": "函數變量乾燥數",
+  "feature.wynntils.dataStorage.dryCount.name": "無神話物品盒數",
```
You might have swapped the descriptions and names of these four lines

Jimmy-sheep replied:
Ahh, I just translate the name part. I'll change the description later.

Commit:
Merge branch 'main' into main

Commit:
[zh_tw] Update translation (Part 7)

Commit:
[zh_tw] Update translation (Part 7.5)

Replace `當前` into `目前`.

Commit:
Merge branch 'main' into main

Commit:
Merge branch 'main' into main

Commit:
[zh_tw] Update translation (Part 8) + fix conflict

Commit:
Merge branch 'main' into main

Jimmy-sheep committed:
ci: spotless formatting

Commit:
[zh_tw] Update translation (Part 8.5)

Fix typo.

Commit:
[zh_tw] Update translation (Part 9)

Replace `記錄點`(Waypoint) into `導航點`.

Commit:
Merge branch 'main' into main

Commit:
Merge branch 'main' into main

Commit:
Merge branch 'main' into main

Commit:
Update zh_tw.json

Fix typo `自訂義` into `自定義`.
Replace `比率` into `層級`.
Replace `打開` into `開啟`.

Jimmy-sheep reviewed (commented):
⇈ [zh_tw] Update translation (Part 10)

Jimmy-sheep committed:
[zh_tw] Update translation (Reviewing Part 1)

Fix some typo.
Re-translate some string.
Fix some punctuation.
Replace `醒目標示` into `醒目提示`.
Replace `該` into `應該`.
Replace `覆蓋` into `覆寫`.

Jimmy-sheep committed:
Merge branch 'main' into main

Jimmy-sheep committed:
[zh_tw] Update translation (Reviewing Part 2) 

Fix something.

Jimmy-sheep commented:
----- Finish Translation -----

Incompleteusern commented:
Nice, once again thank you for doing this.

Jimmy-sheep committed:
Add missing strings

Jimmy-sheep commented:
Btw, I think I should mention here:
The zh_tw and zh_cn can't seen as same language, there's a lot of phraseology are different between Taiwan and China. 
They can just be reference between them.

Jimmy-sheep committed:
Merge branch 'main' into main

Incompleteusern commented:
> Btw, I think I should mention here: The zh_tw and zh_cn can't seen as same language, there's a lot of phraseology are different between Taiwan and China. They can just be reference between them.

Fair enough, that's probably why the original translation makes little sense xd

magicus reviewed (dismissed):
I can't vouch for the content of the translation, but the format looks fine.

Jimmy-sheep committed:
Merge branch 'main' into main

Jimmy-sheep committed:
Merge branch 'main' into main

magicus commented:
@Incompleteusern If you are ok with this, feel free to merge it.

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,1001 @@
+  "feature.wynntils.itemHighlight.craftedHighlightEnabled.name": "製作物品醒目提示",
+  "feature.wynntils.itemHighlight.emeraldPouchHighlightEnabled.description": "啟用綠寶石布袋的醒目提示",
+  "feature.wynntils.itemHighlight.emeraldPouchHighlightEnabled.name": "綠寶石布袋醒目提示",
+  "feature.wynntils.itemHighlight.fabledHighlightColor.description": "Fabled 物品應該要用什麼顏色的醒目提示?",
```
Fabled should be 傳說 here, might have forgot to replace,

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,1001 @@
+  "feature.wynntils.itemHighlight.emeraldPouchHighlightEnabled.name": "綠寶石布袋醒目提示",
+  "feature.wynntils.itemHighlight.fabledHighlightColor.description": "Fabled 物品應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.fabledHighlightColor.name": "傳說物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.fabledHighlightEnabled.description": "啟用 Fabled 物品的醒目提示",
```
Here too

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,1001 @@
+  "feature.wynntils.itemHighlight.hotbarHighlightEnabled.description": "啟用快捷欄中物品的醒目提示",
+  "feature.wynntils.itemHighlight.hotbarHighlightEnabled.name": "快捷欄物品醒目提示",
+  "feature.wynntils.itemHighlight.hotbarOpacity.description": "快捷欄中醒目提示的物品應該使用什麼層級的不透明度?",
+  "feature.wynntils.itemHighlight.hotbarOpacity.name": "快捷欄醒目提示透明度",
```
`透明度` should probably be `不透明度`

Jimmy-sheep replied:
Sorry, my mistake.

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,1001 @@
+  "feature.wynntils.itemHighlight.inventoryHighlightEnabled.name": "物品欄物品醒目提示",
+  "feature.wynntils.itemHighlight.inventoryOpacity.description": "物品欄中醒目提示的物品應該使用什麼層級的不透明度?",
+  "feature.wynntils.itemHighlight.inventoryOpacity.name": "物品欄醒目提示不透明度",
+  "feature.wynntils.itemHighlight.legendaryHighlightColor.description": "Legendary 物品應該要用什麼顏色的醒目提示?",
```
Same comment, for the sake of not posting the same comment, this also occurs a few more times below. Unless you are trying to show the chinese translation in the title but the English variant in the description?

Jimmy-sheep replied:
> ----- Translation Notes -----
> 4. The strings that have proper noun(ex: Item Rarities, Perfect prifix or Item Corkian Amplifier) I only translate the `xxx.name`, and keep `xxx.description` in English to prevent user being confuse, if need translates please let me know.

Yes, I want to use Chinese in `Name` and English in `Description`, if user don't know what tier means in Chinese then they can check in description.

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,1001 @@
+  "feature.wynntils.itemHighlight.rareHighlightColor.name": "稀有物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.rareHighlightEnabled.description": "啟用 Rare 物品的醒目提示",
+  "feature.wynntils.itemHighlight.rareHighlightEnabled.name": "稀有物品醒目提示",
+  "feature.wynntils.itemHighlight.setHighlightColor.description": "套裝物品應該要用什麼顏色的醒目提示?",
```
If you are doing the "unless" portion of the above comment, then `套裝` should be `set` here, same for the description below.

Jimmy-sheep replied:
I only do this in "Item Rarities",such as: `Mythic`, `Fabled`, `Legendary`, `Rare`, `Unique` and `Normal`. And I think `set` in Chinese is clear enough for users. 
Or do you think I should also translate these?

Jimmy-sheep replied:
Also this can make the string prettier, I don't want to do something like this:
`傳奇 (Legendary) 物品醒目提示顏色`
This will make the string too long, and looks duplicate between the word `傳奇` and `Legendary`.

Incompleteusern replied:
Fair enough. I think you should translate `Set` for consistency purposes.

Jimmy-sheep committed:
Fix translate mistake

Jimmy-sheep committed:
Re-translate to setHighlight

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,1001 @@
-  "feature.wynntils.lootrunUtils.undoSuccessful": "撤銷了路徑的一部。",
-  "feature.wynntils.lowHealthVignette.animationSpeed.description": "如果在用脈衝動畫，脈衝的速度是多少？",
+  "feature.wynntils.itemTextOverlay.amplifierTierEnabled.description": "是否應該繪製一個顯示 Corkian Amplifier 品階的物品覆疊層?",
+  "feature.wynntils.itemTextOverlay.amplifierTierEnabled.name": "科金放大器品階格式",
```
Translating ”Corkian" seems a bit odd

Jimmy-sheep replied:
I was thinking about should I translate this or not. If I don't translate, this item will be half is English, half is Chinese, this might look wierd. So I decided to translate the full item name in `xxx.name`, and left the `xxx.description` being English.

Incompleteusern commented on common/src/main/resources/assets/wynntils/lang/zh_tw.json:
```diff
@@ -1,982 +1,1001 @@
+  "feature.wynntils.itemTextOverlay.powderTierRomanNumerals.name": "粉塵品階羅馬數字",
+  "feature.wynntils.itemTextOverlay.powderTierShadow.description": "粉塵品階的覆疊層是否應該繪製陰影?",
+  "feature.wynntils.itemTextOverlay.powderTierShadow.name": "粉塵品階文字陰影",
+  "feature.wynntils.itemTextOverlay.skillIconEnabled.description": "將技能圖示添加到具有元素的物品。",
```
Is there a reason some of these descriptions begin with 將 and others don't?

Jimmy-sheep replied:
Just a different phraseology, I can change it.

Btw I found that I have translate mistake here at `添加`, thanks.

Incompleteusern replied:
Is there a reason this sentence specifically isn't a question?

Jimmy-sheep replied:
This sentence is not a question in en_us.

> `Adds skill icons to items with elements.`

Incompleteusern replied:
Oh, ok then

Jimmy-sheep replied:
Or do you think I should make it become a question sentence?

Incompleteusern replied:
I think the en_us.json is mistaken in that case, but there isn't a need rn until the english version is updated to a sentence ig

Jimmy-sheep committed:
Fix translate mistake

Incompleteusern committed:
Merge branch 'main' into main

Jimmy-sheep committed:
Fix translate mistake

Jimmy-sheep committed:
Re-translate the string

Jimmy-sheep committed:
Merge branch 'main' into main

Jimmy-sheep committed:
Re-translate chatRedirect.xxx.description strings

Incompleteusern committed:
Merge branch 'main' into main

Incompleteusern commented:
I've looked at the first half, and am probably going to merge it as is for now, and finish reading it on my own time

Jimmy-sheep commented:
> I've looked at the first half, and am probably going to merge it as is for now, and finish reading it on my own time

If there's other that need to re-translate, please tell me.  :grinning:
